algorithms.py

```diff
@@ -228,6 +232,90 @@ def add(self, individual):
     def __iter__(self):
         return iter(self._ancestors)
 
+    def find_indv(self, indv):
+        """
+        find the index of an ancestor in population
+        """
+        return self._ancestors.index(indv)
+
+
+class TrajectorySimulator(object):
+    """
+    Class to simulate an allele frequency trajectory on which to condition
+    the coalescent simulation.
+    """
```

Comment:
I've changed the interface of this a bit so that it's now reponsible for simulating trajectories.